metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg_common.py

```diff
@@ -146,19 +147,40 @@ def get_catalog(self) -> Catalog:
         return load_catalog(name=catalog_name, **catalog_config)
 
 
+class TopTableTimings:
```

Comment:
have you considered using the existing `TopKDict`? 
https://github.com/datahub-project/datahub/blob/c6bb65fc8d3cbd10e52e2f64ee808fa6e0265360/metadata-ingestion/src/datahub/utilities/stats_collections.py#L17

Reply:
Yes but it didn't fit my solution in 100%, I wanted to log some more information than just key of the dictionary, I would need to either make dictionary key a tuple (making it even more prone to errors) or keep reference table, which needs to be printed somehow anyway. I decided this would be the simplest solution.